sdks/python/apache_beam/io/datastore/v1/helper.py

```diff
@@ -82,3 +94,148 @@ def str_compare(s1, s2):
     return -1
   else:
     return 1
+
+
+def get_datastore(project):
+  """Returns a Cloud Datastore client."""
+
```

Comment:
There is no need for an empty space between docstring and the first line of the code. (Same for multiple functions in this PR)